sympy/functions/elementary/complexes.py

```diff
@@ -503,6 +503,16 @@ def eval(cls, arg):
             if not unk or not all(conj.has(conjugate(u)) for u in unk):
                 return sqrt(expand_mul(arg*conj))
 
+        # if the expression is a function, then try to see if its positive or negative
+        # functions which have `is_positive` and `is_negative` values of form - bool or None
+        if isinstance(arg, Function) and not isinstance(arg.is_positive,property) \
```

Comment:
The second argument of `isinstance()` should have classinfo. 'property' is not a class.